src/db/AccessRequest.db.ts

```diff
@@ -32,6 +36,44 @@ export async function createRequest(
                 newRequest,
             );
 
+            let emailAddresses: any[] = [];
```

Comment:
I think all this should be in a separately called function. Reason is that it is impossible to cover any of these lines within the transaction with tests because it is a nested function.

Reply:
Yup good point! I was planning on pulling out all of the GC Notify stuff after but might as well do it now :)